realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -83,7 +83,7 @@
                 srcClassName);
     }
 
-    RealmResults(BaseRealm realm, Collection collection, Class<E> clazz) {
+    public RealmResults(BaseRealm realm, Collection collection, Class<E> clazz) {
```

Comment:
Is this really needed?

Reply:
Good catch, not with Mulong's refactor, fixing